templates/circleci/.circleci/config.yml.tmpl

```diff
@@ -2,30 +2,29 @@
 # Make improvements in fogg, so that everyone can benefit.
 {{- $circleCI := . }}
 version: 2
-
 working_directory: &working_directory ~/fogg
 
 jobs:
-  build:
+{{- range $idx, $testBucket := $circleCI.TestBuckets }}
+  terraform_{{$idx}}:
```

Comment:
Curious why circle's built-in parallelism won't work here?

Reply:
I guess they have many forms of built=in parallelism. Are you referring to the command line globbing/splitting?

Reply:
Essentially, yes. The splitting by timings in particular looks pretty useful.

Reply:
It would work. We can make that improvement later?